src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -328,15 +336,15 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
 
               <TokenExtras>
                 {amountValue && <StyledText>{formatUsd(!loading && !inputError ? amountValue : '0')}</StyledText>}
-                {yieldPercent && (
-                  <StyledText>
-                    {t('components.transaction.token-input.yield')} <ContrastText>{yieldPercent}</ContrastText>
-                  </StyledText>
-                )}
```

Comment:
Are we sure we do not want to display the yield on the component? When this component is displayed as a modal it might be useful for the user, specially since we allow users to change vaults if the modal is opened through the portfolio page.

If we decide to remove it, we should refactor this component and remove the `yieldPercent` prop, since its no longer needed.

Reply:
I would leave that for another PR as well, this one should just be to add the Zap text

Reply:
was asked to remove this as it's duplicate info 
can we add conditional display? i'll remove this for now and add to another PR

![image](https://user-images.githubusercontent.com/80323528/169620865-6b35cb41-20fa-4d1e-9e52-ff414149f3f5.png)

Reply:
Yeah, its seems to be duplicated when its opened directly through the vaults detail page. Might be worth just leaving it when it opens as a modal from other pages.